website/static/js/project.js

```diff
@@ -61,38 +60,8 @@ NodeActions.forkNode = function() {
                 isCors: true,
                 data: payload
             }
-        ).done(function(response) {
-            $osf.unblock();
-            bootbox.confirm({
-                message: '<h4 class="add-project-success text-success">Fork created successfully!</h4>',
-                callback: function(result) {
-                    if(result) {
-                        window.location = response.data.links.html;
-                    }
-                },
-                buttons: {
-                    confirm: {
-                        label: 'Go to new fork',
-                        className: 'btn-success'
-                    },
-                    cancel: {
-                        label: 'Keep working here'
-                    }
-                },
-                closeButton: false
-            });
-        }).fail(function(response) {
-            $osf.unblock();
-            if (response.status === 403) {
-                $osf.growl('Sorry:', 'you do not have permission to fork this project');
-            } else if (response.status === 504) {
-                $osf.growl('Sorry:', 'This is taking longer than normal. </br>' +
-                    'Please check back later to access your new fork and if still unavailable, contact support@cos.io');
-            } else {
-                $osf.growl('Error:', 'Forking failed');
-                Raven.captureMessage('Error occurred during forking');
-            }
-        });
+        );
+        $osf.growl('Fork status', 'Your fork is being forked you\'ll get an email when it\'s complete.', 'info');
```

Comment:
I don't know, does everything product proposed get a demo? This seems mostly as though it's removing stuff, instead of modals you get a growl box then an email, I need more clarity on what gets demoed because I am waiting 1 to 2 weeks every time something is demoed and so I try to avoid it unless it's specifically in the ticket.